server/GQLFactory/helpers/typeHelpers.js

```diff
@@ -73,10 +73,47 @@ mutationsHelper.mutationFields = (
 
 /*   Functions facilitating creation of custom types */
 const customHelper = {};
+/*  Loops through SQL columns getting their name as fieldName, type, and isNullable to be returned as fields */
+customHelper.getFields = (primaryKey, foreignKeys, columns) => {
+  let fields = ``;
 
-customHelper.getFields = () => {};
+  for (const fieldName of Object.keys(columns)) {
+    if (!(foreignKeys && foreignKeys[fieldName]) && fieldName !== primaryKey) {
```

Comment:
! operator outside of an AND statement seems confusing. Is there a better way to handle this?

Reply:
going to change syntax as well !foreignKeys || !foreignKeys[fieldName]...